app.py

```diff
@@ -12,7 +12,7 @@
 
 app.include_router(auth.router)
 app.include_router(chats.router)
-app.include_router(points.router)
+#app.include_router(points.router)
```

Comment:
I think you can safely delete this router entirely along with the corresponding file